connector/src/main/scala/com/microsoft/kusto/spark/utils/CslCommandsGenerator.scala

```diff
@@ -5,19 +5,19 @@ import com.microsoft.kusto.spark.datasink.KustoWriter.TempIngestionTablePrefix
 object CslCommandsGenerator{
 
   // Not used. Here in case we prefer this approach
-  def generateFindOldTemporaryTablesCommand2(database: String): String = {
+  private[kusto] def generateFindOldTemporaryTablesCommand2(database: String): String = {
     s""".show database $database extents metadata | where TableName startswith '$TempIngestionTablePrefix' | project TableName, maxDate = todynamic(ExtentMetadata).MaxDateTime | where maxDate > ago(1h)"""
   }
 
-  def generateFindOldTempTablesCommand(database: String): String = {
+  private[kusto] def generateFindOldTempTablesCommand(database: String): String = {
     s""".show journal | where Event == 'CREATE-TABLE' | where Database == '$database' | where EntityName startswith '$TempIngestionTablePrefix' | where EventTimestamp < ago(1h) and EventTimestamp > ago(3d) | project EntityName """
   }
 
-  def generateFindCurrentTempTablesCommand(prefix: String): String = {
+  private[kusto] def generateFindCurrentTempTablesCommand(prefix: String): String = {
```

Comment:
why?
this can be open for users who want to run queries on java client ..
and anyway why did u leave some as public ?

Reply:
I don't think we should allow Java client dependencies on the connector.
On the second - missed entries added in the merge, will fix. thanks

Reply:
Thinking again... I just made the whole object private[kusto]